packages/pages/src/common/src/function/internal/types.test.ts

```diff
@@ -92,23 +109,33 @@ describe("internal/types - convertFunctionModuleToFunctionModuleInternal", () =>
         false
       );
     const expected = {
-      default: "exampleFunction",
       config: {
```

Comment:
I keep getting confused by the property names we have here
```
{
      config: {
        name: "Test Function",
        functionName: "exampleFunction",
        event: "API",
      },
      path: "src/functions/example.ts",
      filename: "example.ts",
      slug: "myFunction",
    };
```
because `path` really means path to the literal file while slug is the output of getPath. Can you update `path` to `filePath`?